array.rb

```diff
@@ -45,15 +45,22 @@ def each
   end
 
   # call-seq:
-  #    array.shuffle!(random: Random) -> array
+  #   shuffle(random: Random) -> new_array
```

Comment:
This is documentation for `Array#shuffle!`, not `Array#shuffle`.

```suggestion
  #   shuffle!(random: Random) -> new_array
```